### PR TITLE
Shifted stringify_and_store_api_record to common utils

### PR DESCRIPTION
**Pull request by MueezKhan246**
Description:
Shifted stringify_and_store_api_record to common utils.

JIRA:
https://2u-internal.atlassian.net/jira/software/c/projects/ENT/issues/ENT-8277

### CODE REVIEW

File: integrated_channels/utils.py [hunk 1]
```diff
@@ -543,3 +543,42 @@ def store_api_call(
...
+                enterprise_customer=enterprise_customer,
+                enterprise_customer_configuration_id=enterprise_customer_configuration_id,
+                endpoint=endpoint,
+                payload=data,
+                time_taken=time_taken,
+                status_code=status_code,
+                response_body=response_body,
+            )
+        except Exception as e:   # pylint: disable=broad-except
+            print(f"Failed to store data in the database: {e}")
```
Comment (sameenfatima78) [inline integrated_channels/utils.py:584]: do we have unit tests for this util?

**Review by sameenfatima78 — APPROVED** [review]
Looks good just one comment.

File: integrated_channels/utils.py (hunk 1 above)
Comment (hamzawaleed01) [reply integrated_channels/utils.py:584]: nope, let me add it quickly!!

Comment (MueezKhan246) [reply integrated_channels/utils.py]: the test case for stringify method looks good to me

**Review by hamzawaleed01 — APPROVED** [review]

**Pull request merged**